docs/DeveloperGuide.adoc

```diff
@@ -294,72 +296,225 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+*Extensions*
+
+[none]
+* 1a. NOVA cannot find the contact in the contact list
++
+[none]
+** 1a1. NOVA informs student that the contact to delete does not exist
++
+Use case ends.
+
+[discrete]
+=== Use case 2: Delete contact's picture
+
+*MSS*
+
+1.  Student enters deletepic command with the contact’s name
+2.  NOVA deletes the contact's picture
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The contact does not have any picture uploaded
++
+[none]
+** 1a1. NOVA informs student that there is no picture to delete
++
+Use case ends.
+
+[discrete]
+=== Use case 3: Delete an event
+
+*MSS*
+
+1. Student enters delete command with details of the event
+2. NOVA deletes the event
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. NOVA cannot find the event given
+** 1a1. NOVA informs student that the event does not exist
++
+Use case ends.
+
+[discrete]
+=== Use case 4: Add notes to a project task in project section of progress tracker
+
+*MSS*
+
+1.  User enter command to add notes to the project task.
+2.  Progress tracker adds notes to the project task.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. No such project task exist.
+*   1a1. NOVA shows an error message.
++
+Use case ends.
 
 [discrete]
-=== Use case: Delete person
+=== Use case 5: User edits a note.
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User enter command to edit note.
+2.  Progress tracker replace old note with new note.
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 1a. No prior note was added.
+*   1a1. NOVA adds edit as new note.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+[discrete]
+=== Use case 6: User deletes a note.
+
+*MSS*
+
+1.  User enter command to delete note.
+2.  Progress tracker deletes note.
 +
+Use case ends.
+
+*Extensions*
+
 [none]
-** 3a1. AddressBook shows an error message.
+* 1a. Note to be deleted does not exist.
+*   1a1. NOVA shows error message.
 +
-Use case resumes at step 2.
+Use case ends.
+
+[discrete]
+=== Use case 7: View the timetable
+
+*MSS*
+
+1. User requests for the timetable
+2. NOVA shows the timetable
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The timetable is empty.
++
+Use case ends.
 
 _{More to be added}_
 
 [appendix]
 == Non Functional Requirements
 
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  The application should respond to every command within one second.
+.  Technical requirements: The application should work on both 32-bit and 64-bit environments.
```

Comment:
Given that almost all personal computers are running 64-bit OSes, is this a necessary constraint?